---
title: Research in the JBD Lab
layout: default
group: research
---

<div class="row">
# Our research
<div align="justify">
Bacteriophages (phages) are the most abundant biological entities on the planet and can be viewed as professional manipulators of their host bacteria. In our group, we study how defense mechanisms encoded by bacteria target phages and how phages co-evolve to deal with that pressure. These bacterial “immune systems,” including CRISPR-Cas and restriction enzymes, are among the many fundamental discoveries and biotechnological tools uncovered by studying phage biology. The central mission of the lab is to discover and characterize the molecular determinants that drive phage infection of bacterial hosts. We study bacterial biology through a phage lens. We use a combination of genetic, molecular and biochemical approaches to characterize the arms race between bacteria and phages, with a goal to better understand microbial ecosystems. Furthermore, we hope to make discoveries that will be influential in combatting infectious disease through phage therapy and in developing novel biotechnologies.
</div>
<br>
<br>
</div>

### Phage-encoded anti-CRISPRs
<div class="row">
<div class="col-md-6 order-md-1">
<div align="justify">
The battle between bacteria and phages has led to the evolution of multiple phage resistance mechanisms such as CRISPR-Cas and subsequent counter-resistance mechanisms employed by the phage. While a graduate student in Alan Davidson’s lab, Joe discovered and characterized the first examples of phage-encoded proteins that inhibit CRISPR function, called anti-CRISPRs. These diverse proteins directly bind to and antagonize different CRISPR associated proteins, thus blocking phage targeting. We have focused more recently on understanding how anti-CRISPR proteins are deployed and regulated during phage infection. Additionally, we are interested in understanding why there are so many distinct anti-CRISPRs in closely related phages and what the costs and benefits are to possessing different ones. Finally, we have pioneered new methods for discovery of anti-CRISPR proteins that inhibit CRISPR-Cas9 and Cas12 using bioinformatics and experimentation. In addition to providing valuable new tools for the gene editing community, these discoveries pave the way to understanding why and when CRISPR-Cas fails in nature.
</div>
</div>
<div class="col-md-6 order-md-2 align-self-center">
<img class="img-fluid" src="/static/img/pub/marino_2020.png">
<!-- <a href="http://www.ucsf.edu"><img class="inline-block navb-icon" src="/static/img/ucsf_logo_white.svg" alt="University of California, San Francisco (UCSF) logo"></a> -->
</div>
</div>

### CRISPR-Cas regulation
<div class="row">
<div class="col-md-9 order-md-2 ">
<div align="justify">
The early findings that a CRISPR array and the associated Cas genes could be transferred into a heterologous bacterial system and still be functional told us that, in general, these systems are autonomous. Further, with the successful transfer of some CRISPR-Cas systems into human cells and animals, it is quite clear that we understand the basic requirements for CRISPR function. What is poorly understood, however, is how CRISPR-Cas systems are regulated in their native hosts. What are the physiological cues that bacteria receive that can control CRISPR expression? We extensively use Pseudomonas aeruginosa as a model system for Type I CRISPR-Cas systems, but we know nothing about how the factors that control these systems are expressed. This is an especially intriguing question due to a rich literature describing P. aeruginosa possessing more regulatory systems than most bacteria, presumably to equip it for its ‘generalist’ lifestyle, being both a ubiquitous microbe in the environment and a highly drug resistant opportunistic human pathogen. We have identified CRISPR regulatory pathways that inversely controls alginate, a common biofilm polysaccharide. This suggests that surface lifestyle controls CRISPR-Cas expression. We also identified a pirated CRISPR-Cas repressor protein that is encoded by phages and other mobile elements.
</div>
</div>
<div class="col-md-3 order-md-1 align-self-center">
<img class="img-fluid" src="/static/img/pub/borges_2020.png">
</div>
</div>

### CRISPR-Cas3 genome editing in microbes and phages
<div class="row">
<div class="col-md-7 order-md-1">
<div align="justify">
CRISPR-Cas systems were functionally characterized just 13 years ago as bacterial immune systems that target bacteriophages. Since then, there has been an explosion of interest in this system for its widespread presence in the microbial world as well as its facile programmability. This has formed the basis of a revolutionary and Nobel prize winning gene editing technique, CRISPR-Cas9. However, Cas3 enzymes are natural effectors for Type I CRISPR-Cas system that harness the benefit of programmability, with the upside or large genomic deletions. We have harnessed the Type I CRISPR-Cas3 effector for making large deletions in bacterial genomes, with specified (via homology directed repair) or unspecific boundaries. We are using this technology to support other discovery efforts in the lab and working on new applications for Cas3. We are also applying this enzyme and others to phage engineering efforts, in the hopes of making it possible to enable genetic tractability for any phage.
</div>
</div>
<div class="col-md-5 order-md-2 align-self-center">
<img class="img-fluid" src="/static/img/pub/csorgo_2020.png">
</div>
</div>

### Bacterial pan-immunity
<div class="row">
<div class="col-md-6 order-md-2">
<div align="justify">
Historically, we have been focused on studying CRISPR-Cas systems in their natural settings, specifically understanding how these systems are regulated and inhibited/evaded by phages. More recently, we have undertaken efforts to study other bacterial immune systems that block phage replication, to better understand their mechanisms and importance for microbial and phage biology. We are focusing on endogenous immune function in our two favorite model human pathogens Pseudomonas aeruginosa and Listeria monocytogenes.
</div>
</div>
<div class="col-md-6 order-md-1 align-self-center ">
<img class="img-fluid" src="/static/img/pub/huiting_2021.png">
</div>
</div>

### Phage "nucleus-like" structure
<div class="row">
<div class="col-md-6 order-md-2 align-self-center">
<img class="img-fluid" src="/static/img/pub/guan_2020.png">
</div>
<div class="col-md-6 order-md-1">
<div align="justify">
Recent screening efforts in our lab have identified lytic jumbo phages that infect Pseudomonas aeruginosa and are resistant to CRISPR-Cas and restriction enzyme targeting. These phages assemble a remarkable proteinaceous nucleus-like structure during infection, which we have shown is causal for immune evasion. Other phage families also show CRISPR evasion phenotypes, which we are characterizing further. Jumbo phages from the phiKZ family are naturally able to evade destruction by all DNA-targeting immune systems, suggesting that the phage genome doesn’t get exposed to cytoplasmic enzymes from the start of infection, to the end. We are currently focused on dissecting the early events after infection that enable resistance to immune systems and discovering new mechanisms by which Pseudomonas aeruginosa fights back against this phage. As a potent killer of P. aeruginosa, this phage is a strong candidate for phage therapy and is being heavily studied by many labs with this focus. We hope to provide basic molecular insights and phage engineering tools to help this cause.
</div>
</div>
